### PR TITLE
Minor space optimizations for Dockerfile

### PR DESCRIPTION
**Pull request by sethforprivacy**
Automatically updates via apt and installs minimal dependencies w/ clearing of cache directories after apt dependency installation.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,16 +8,14 @@ MAINTAINER Igor Cota <igor@foundationdevices.com>
 
 WORKDIR /root
 
-ARG DEBIAN_FRONTEND=noninteractive
```
Comment (icota) [inline Dockerfile:11]: Does this really help? I thought `ARG` doesn't add layers?

Comment (sethforprivacy) [reply Dockerfile:11]: Sorry, meant line optimization, not layers!

Comment (sethforprivacy) [reply Dockerfile:11]: It's just a preference, happy to drop that change.

Comment (icota) [reply Dockerfile:11]: Last thing Dockerfiles need is less lines @sethforprivacy!

Please drop it and I'll merge the rest

Comment (sethforprivacy) [reply Dockerfile:11]: Done!

**Review by icota — APPROVED** [review]

**Pull request merged**